components/Footer.scss

```diff
@@ -0,0 +1,79 @@
+@import '../styles/variables.scss';
+
+footer {
+  background: $primary;
+  color: $secondary;
+  height: 100%;
+  width: 100vw;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  margin: 0;
+}
+
+.footer-wrapper {
```

Comment:
Since we're using SCSS here look to nest your elements and use the "&" selector, which acts as the parent class.

So & here, under footer, would be .footer.

`footer {
      &--wrapper {
            &__left,
            &__right
      }
}`

Reply:
Should all be nested now :)